pincer/commands/commands.py

```diff
@@ -774,130 +728,87 @@ async def __remove_unused_commands(self):
         Remove commands that are registered by discord but not in use
         by the current client
         """
-        registered_commands = list(
+        local_registered_commands = list(
             map(
                 lambda registered_cmd: registered_cmd.app,
                 ChatCommandHandler.register.values(),
             )
         )
-        keep = []
 
-        def predicate(target: AppCommand) -> bool:
-            for reg_cmd in registered_commands:
-                reg_cmd: AppCommand = reg_cmd
-                if target == reg_cmd:
+        def should_be_removed(target: AppCommand) -> bool:
+            for reg_cmd in local_registered_commands:
```

Comment:
cannot that be converted to 
```py
return All(
    (
        target.name =! reg_cmd.name
        and target.guild_id == reg_cmd.guild_id
    ) for reg_cmd in local_registered_commands
)

```
not sure about it and the benefits tho

Reply:
I don't think this is more readable

Reply:
Is agree with Lunar here about the readability